maintainers/scripts/update.py

```diff
@@ -77,7 +78,9 @@ async def run_update_script(nixpkgs_root: str, merge_lock: asyncio.Lock, temp_di
         eprint(f"--- SHOWING ERROR LOG FOR {package['name']} ----------------------")
         eprint()
         eprint(e.stderr.decode('utf-8'))
-        with open(f"{package['pname']}.log", 'wb') as logfile:
+        with open(f"{package['pname']}.out.log", 'wb') as logfile:
```

Comment:
@jtojnar would you mind taking a look at this commit?  It was useful for me to debug dotnet stuff which often writes to stdout.

I had originally omitted the .log extension, but it's in `.gitignore`, so I decided to keep it.

I think ideally this would write both streams to a single temporary log file, but that's likely a bigger change.  I can drop this commit if anyone has any concerns.

Reply:
I dropped this change because there's no sense rushing it or holding up this PR.  It would still be nice to get some feedback on dealing with update scripts that write to both stdout and stderr.